recipe/meta.yaml

```diff
@@ -93,6 +93,7 @@ test:
     - test -f ${PREFIX}/lib/libMOAB.so  # [linux]
     - test -f ${PREFIX}/lib/libMOAB.dylib  # [osx]
     - python examples/python/laplaciansmoother.py MeshFiles/unittest/surfrandomtris-4part.h5m 25
+    - mbtempest --help  # [tempest == 'tempest']
```

Comment:
Make sure we actually got TempestRemap when we asked for it.

Reply:
You should invoke `${PREFIX}/bin/mbtempest` here, no ?

Reply:
> You should invoke ${PREFIX}/bin/mbtempest here, no ?

No, we want to make sure it's in the path once you install it in the test environment. 

>  You could do: make -C test imoab_remapping && test/imoab_remapping and make -C test/parallel imoab_coupler && mpiexec -n 2 test/parallel/imoab_coupler for example.

Are these tests are run during build?  If so, that is a separate phase and with a separate environment from this testing.   

During the testing phase, we are not in the same place as where the code was built and in a completely different environment.  This is similar to if you ran:
```
conda create -y -n test moab
```
and then ran these tests.  So the make file you are trying too access with the above examples would not be available.  If these tests aren't already invoked, they should be added to the build script instead.